pysilcam/tests/test_process.py

```diff
@@ -101,4 +162,4 @@ def test_output_files():
     #os.makedirs(reportdir, exist_ok=True)
```

Comment:
Remove unused code?

Reply:
I am handling this on another branch atm.